test/google-calendar-api.spec.js

```diff
@@ -13,64 +13,117 @@ var googleApiWrapper = rewire('../src/google-calendar-api');
+					expect(fakeCalendarApi.list.firstCall.args[0].calendarId).to.equal(clientEmail)
+				);
+
+				it('should set the "timeMin" to event date', () =>
+					expect(moment(fakeCalendarApi.list.firstCall.args[0].timeMin).isSame(moment(date), 'day')).to.equal(true)
+				);
+
+				it('should set the "timeMax" parameter to day after event date', () =>
+					expect(moment(fakeCalendarApi.list.firstCall.args[0].timeMax).isSame(moment(date).add(1, 'day'), 'day')).to.equal(true)
+				);
+
+				it('should find that event ID', () =>
+					expect(resolvedResult).to.eventually.equal(eventId)
+				);
+			});
+
+			describe('And there is not already a WFH event', () => {
+				beforeEach(() => {
+					apiResponse.items = [{ id: chance.string(), summary: `${ chance.name({ middle: true }) } - WFH` }];
+					return act();
+				});
+
+				it('should not find that event', () => {
+					expect(resolvedResult).to.eventually.not.be.ok
+				});
+			});
 		});
+		describe('And we want to see if a WFH event exists in a time interval', () => {
+			var apiResponse, employeeName, resolvedResult, startDateTime, endDateTime;
+			var act = () => resolvedResult = googleApiWrapper.checkIfWfhEventExists(employeeName, startDateTime, endDateTime);
 
-		describe('And there is already an event with a summary matching the event template like "Person - WFH"', () => {
+			beforeEach(() => {
+				apiResponse = {};
+				employeeName = chance.name();
+				fakeCalendarApi.list = sinon.stub().callsArgWith(1, null, apiResponse);
+			});
+			describe('And there is already an event with a summary matching the event template like "Person - WFH"', () => {
 			var eventId;
 
 			beforeEach(() => {
 				eventId = chance.string();
 				apiResponse.items = [{ id: eventId, summary: `${ employeeName } - WFH` }];
+				startDateTime = new Date(chance.date({ year: 2017 }));
+				endDateTime = new Date(chance.date({ year: 2017 }));
 				return act();
 			});
 
-			it('should pass in the GOOGLE_CLIENT_EMAIL environmental variable as the "calenderId"', () =>
+			it('should pass in the GOOGLE_CLIENT_EMAIL environmental variable as the "calendarId"', () =>
 				expect(fakeCalendarApi.list.firstCall.args[0].calendarId).to.equal(clientEmail)
 			);
 
-			it('should set the "timeMin" to today', () =>
-				expect(moment(fakeCalendarApi.list.firstCall.args[0].timeMin).isSame(moment(), 'day')).to.equal(true)
+			it('should set the "timeMin" to the start time', () =>
+				sinon.assert.match(moment(fakeCalendarApi.list.firstCall.args[0].timeMin).format(), moment(startDateTime).format())
 			);
 
-			it('should set the "timeMax" parameter to tomorrow', () =>
-				expect(moment(fakeCalendarApi.list.firstCall.args[0].timeMax).isSame(moment().add(1, 'day'), 'day')).to.equal(true)
+			it('should set the "timeMax" parameter to the end time', () =>
+				sinon.assert.match(moment(fakeCalendarApi.list.firstCall.args[0].timeMax).format(), moment(endDateTime).format())
```

Comment:
Also, since you have two `moment` objects, I think you can compare them outright instead of having to `format()` then and compare the strings.